common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1559,6 +1569,19 @@ def can_toggle_course_pacing(self):
         if not self.start:
             return False
         return datetime.now(utc) <= self.start
+    
+    @property
+    def enable_discussion(self):
```

Comment:
`enable_discussion` sounds like an action, and might be good for the name of a method/function that enabled discussion, not for a flag. 
That said, this property is superfluous and should be removed.